yarn.lock

```diff
@@ -1102,6 +1102,50 @@
     exec-sh "^0.3.2"
```

Comment:
It looks like `yarn install` needs to be re-run, I'm not seeing the dependencies of jest-puppeteer removed.